examples/mutated_rhinovirus_c.yaml

```diff
@@ -0,0 +1,153 @@
+# A readable name for the benchmark
+description: List of genomes with different mutation amount to a Rhinovirus C
+# Number of reads per organism
+reads_per_organism: 10000
+# The sequencer model to emulate (determines the error model used by iss)
```

Comment:
curious: what's iss?

Reply:
`InSilicoSeq` the tool that actually creates the benchmark by simulating results from a particular type of sequencer.